### ISSUE TITLE
add start/stop scripts for display selecta (Trac #181)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#181]
add start/stop scripts for display selecta + add correspondig lines in kill-jebsystem-dead

Migrated from https://code.icecube.wisc.edu/ticket/181
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T22:14:11", 
    "description": "add start/stop scripts for display selecta + add correspondig lines in kill-jebsystem-dead", 
    "reporter": "tschmidt", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547244851732371", 
    "component": "jeb + pnf", 
    "summary": "add start/stop scripts for display selecta", 
    "priority": "minor", 
    "keywords": "JEB, display selecta", 
    "time": "2009-11-04T20:23:52", 
    "milestone": "", 
    "owner": "tschmidt", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20120525T14:01:09: 
* blaufuss changed resolution from "" to "wontfix"
* blaufuss commented:

> needs to be rethought as part of I3Live/I3TV event display...


* blaufuss changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T22:13:37: 
* nwhitehorn commented:

> In changeset 169350:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169350"
> Attempt to fix #181.  You can't use regular readlink becuase it isn't
> everywhere, on the mac it doesn't take 'canonicalize', etc.  pain in
> the ass, python to the rescue as usual.
> ```


* nwhitehorn changed resolution from "wontfix" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T22:14:11: nwhitehorn commented:

> In changeset 169356:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169356"
> Undo of r96 that was supposed to fix #181...   causes problem on network at wisc.
> Thanks to davino and marius for the heads up.
> ```
